### ISSUE TITLE
Free-floating battery from destroyed batteries

### ISSUE THREAD

**Issue by githubuser343** [CleverRaven/Cataclysm-DDA#39470]
# Describe the bug

Turret killed a zombie and it destroyed a battery, leaving behind 300 free-floating battery objects, the supposedly unloadable part of a battery.

# Steps To Reproduce

1. Have a turret shoot zombies
2. Wait until you see a battery object on the ground

# Expected behavior

Battery load to be deleted upon battery destruction, scrap/acid/components left over.

# Versions and configuration

0.E no mods.

# Additional context

I didn't try picking it up.

**Comment by githubuser343** [comment]
![Untitled](https://user-images.githubusercontent.com/62487161/79239109-75710700-7e70-11ea-8b29-77f2f72ec31f.png)

It seems to happen quiet reliably in single level labs.

**Comment by hexagonrecursion** [comment]
Is it possible that #42053 is the actual cause of this?

**Comment by int-ua** [comment]
Removing the `Bug` label until a fresh confirmation.

**Comment by OliveChukar** [comment]
Tested in ver 0.E-9072-g1bb53ba-dirty with core content pack dark days ahead only. Spawned 100 light disposable batteries then shot them with an M4A1 until they were all destroyed checking for free charge goop between every shot. Did not see any charge outside the batteries. Issue not confirmed.

**Comment by int-ua** [comment]
Please comment if this issue is still relevant and should be reopened.

**Comment by LeahLuong** [comment]
@int-ua #48147 seems to have some possible reproduction cases. I came across this myself recently in 0.F-3:

![Screenshot_20220228-094636](https://user-images.githubusercontent.com/91638992/156108682-24972ccb-2c5c-4951-ace9-f4cd73a1b5cb.png)

![Screenshot_20220228-094730](https://user-images.githubusercontent.com/91638992/156108751-38f9c024-69eb-4068-b849-c3363e1be40f.png)

![Screenshot_20220228-094805](https://user-images.githubusercontent.com/91638992/156108794-18cf094b-30b4-4b2c-a135-9af01be61732.png)

I found it on a frequently-traveled road & I'm unsure what it came from. Definitely a Z I killed there dropped something & I ran over it a few times in my travels.
